test/metabase/notification/payload/impl/card_test.clj

```diff
@@ -0,0 +1,91 @@
+(ns metabase.notification.payload.impl.card-test
```

Comment:
have a basic e2e test here, we'll need to convert all tests from `metabase.pulse.send-test` that have all alert tests to this namespace